app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i.freeze
+
+  validates :name, presence: true,
+    length: {maximum: Settings.user.name.max_length_255,minimum: Settings.user.name.min_length_3}
```

Comment:
```suggestion
    length: {maximum: Settings.length.max_length_255, minimum: Settings.length.min_length_3}
```
đổi code trong file setting đi nhé, ri để còn tái sử dụng

Reply:
Dạ chị